src/spxRender.js

```diff
@@ -78,11 +84,64 @@ function renderMixedTextToString(text, suppressWarnings) {
             }
         } else {
             bit = htmlEscape(bit.replace("\\$", "$"));
-            // And now this function is terribly named...
-            bits[i] = preprocessText(bit);
+
+            // Replace Latex tags such as \textbf{...} and \textit{...} with
+            // HTML tags such as <b>...</b> and <i>...</i>
+            bit = processLatex(bit, tagStack);
+            bits[i] = bit;
         }
     }
-    return bits.join('');
+    return preprocessText(bits.join(''));
+}
+
+/**
+ * Replaces Latex sections of a bit of a string with equivalent HTML elements.
+ * At present \textbf{ ... } is replaced with <b> ... </b> and |textit{ ... }
+ * is replaced with <i> ... </i>
+ * @param bit The bit of the string to be modified
+ * @param tagStack A stack of the open tags in the string (to allow a tag
+ * opened in one bit to be closed in a following bit, preserving the
+ * opening/closing order)
+ * @returns {*} The modified bit of a string
+ */
+function processLatex(bit, tagStack) {
```

Comment:
I think this function is badly named - I dont necessarily have a good name for it though!

also is there a reason the tagStack is passed as an argument here?